Gemfile.lock

```diff
@@ -11,89 +11,105 @@ PATH
   specs:
     vulneruby_engine (0.1.0)
       grape (~> 1.5.3)
-      rails (~> 6.0.3, >= 6.0.3.1)
-      rake (~> 12.0)
+      rake (~> 12.3.0)
+      sidekiq (~> 6.3, >= 6.3.1)
+      sidekiq-status (~> 2.1)
       sinatra (~> 2.0)
+      sprockets-rails
```

Comment:
`sprocket-rails` was removed as a default gem in Rails 7, hence the addition of this explicitly.